AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/max/RewardedAdActivity.kt

```diff
@@ -21,7 +23,7 @@ import java.util.concurrent.TimeUnit
 class RewardedAdActivity : BaseAdActivity(),
         MaxRewardedAdListener, MaxAdRevenueListener {
     private lateinit var rewardedAd: MaxRewardedAd
-    private var retryAttempt = 0.0
+    private var retryAttempt: Int = 0
```

Comment:
We usually use type inference in Kotlin and Swift (less code, more "modern"), so `private var retryAttempt` should be good enough.